applications/dds_video/README.md

```diff
@@ -0,0 +1,96 @@
+# DDS Video Application
+
+The DDS Video application demonstrates how video frames can be written to or
+read from a DDS databus in order to provide flexible integration between
+Holoscan applications and other applications (using Holoscan or not) via DDS.
+
+The application can be run as either a publisher or as a subscriber. In either case,
+it will use the [VideoFrame](../../operators/dds/video/VideoFrame.idl) data topic
+registered by the `DDSVideoPublisherOp` or `DDSVideoSubscriberOp` operators in order
+to write or read the video frame data to/from the DDS databus, respectively.
+
+When run as a publisher, the source for the input video frames will come from an
+attached V4L2-compatible camera via the `V4L2VideoCaptureOp` operator.
```

Comment:
Would it be possible to have a replay option in the future?

Reply:
Sure, I don't see why not. Anything that provides `VideoBuffer` objects should work for the source.